trilby-cli/flake.nix

```diff
@@ -1,7 +1,8 @@
 {
   nixConfig = {
     extra-substituters = "https://cache.ners.ch/haskell";
-    extra-trusted-public-keys = "haskell:WskuxROW5pPy83rt3ZXnff09gvnu80yovdeKDw5Gi3o=";
+    extra-trusted-public-keys =
```

Comment:
I’m guessing this entire file got reformatted by accident and there’s no actual diff intended.